I/ImageSegmentation/Compat.toml

```diff
@@ -14,11 +16,15 @@ ImageFiltering = "0.1.4-0.5"
 LightGraphs = "0.9.2-1.2"
 StaticArrays = "0.6-0.10"
 
-["0.0.2-1.1"]
+["0.0.2-1.2"]
 DataStructures = "0.7.1-0.15"
 ImageFiltering = "0.1.3-0.5"
 LightGraphs = "0.9-1.2"
 StaticArrays = "0.5-0.10"
 
 ["1.0-1.1"]
 julia = "0.7-1.3"
+
+["1.2"]
+Images = "0.9-0.18"
+julia = "1.0-1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.